--- hotdoc-0.16/hotdoc/extensions/c/c_extension.py.orig	2024-03-13 06:16:15.409235888 +0100
+++ hotdoc-0.16/hotdoc/extensions/c/c_extension.py	2024-03-13 06:28:20.782664346 +0100
@@ -96,11 +96,11 @@ def get_clang_headers():
         ['llvm-config', '--version']).strip().decode()
     prefix = subprocess.check_output(
         ['llvm-config', '--prefix']).strip().decode()
+    libdir = subprocess.check_output(
+        ['llvm-config', '--libdir']).strip().decode()
     versions = (version, version.split('.').pop(0))
-    for (ver, lib) in itertools.product(
-            versions,
-            ['lib', 'lib64']):
-        p = os.path.join(prefix, lib, 'clang', ver, 'include')
+    for ver in versions:
+        p = os.path.join(libdir, 'clang', ver, 'include')
         if os.path.exists(p):
             return p
 
