--- hotdoc-0.16/setup.py.orig	2023-11-09 15:08:48.000000000 +0100
+++ hotdoc-0.16/setup.py	2024-03-13 19:08:01.337366694 +0100
@@ -361,7 +361,8 @@ class FlexExtension (Extension):
         src_dir = os.path.dirname(self.__flex_sources[0])
         built_scanner_path = src(os.path.join(src_dir, 'scanner.c'))
 
-        self.sources.append(built_scanner_path)
+        if built_scanner_path not in self.sources:
+            self.sources.append(built_scanner_path)
         if newer_group(self.__flex_sources, built_scanner_path):
             cmd = ['flex', '-o', built_scanner_path]
             for s in self.__flex_sources:
